javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,64 @@
+//Save a note
+const notes = [];
+
+function saveNote(content, id, important) {
+  const note = {
+    content: content,
+    id: id,
+    important: important,
```

Comment:
Great, you count also fx. set the important: to a boolean value, since later you are checking whether it equals true. That allows you later to shorten your if statement to 

if(notes[i].important){
rest of your code
}

Reply:
Thank you for the feedback and cool idea :)